developers/academy/zero_to_mvp/101_hello_weaviate/15_overview_vectors.mdx

```diff
@@ -9,56 +9,57 @@ import ReactPlayer from 'react-player/lazy'
 
-Weaviate enables vector searches by indexing and storing data objects and corresponding vector embeddings from machine learning models.
+The vector representation of an object's meaning is called a "vector embedding".
+
+Weaviate enables vector searches by indexing and storing data objects and their corresponding vector embeddings. The vector embeddings come from machine learning models.
 
-In plain terms, Weaviate processes and organizes your data in such a way that objects can be retrieved based on their similarity to a query. In order for it to perform these tasks at speed, Weaviate does two things that traditional databases do not. They are:
+In plain terms, Weaviate processes and organizes your data in such a way that objects can be retrieved based on their similarity to a query. To perform these tasks at speed, Weaviate does two things that traditional databases do not. Weaviate:
 
-- Quantifying similarity, and
-- Indexing vector data
+- Quantifies similarity
+- Indexes vector data
 
-These aspects enable Weaviate to do what it does.
+These operations enable Weaviate to do what it does.
 
 ### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp;Quantifying similarity
 
 As we've mentioned, vector searches are similarity-based, but what does that actually mean? How do we determine that two pieces of data are "similar"? What does it mean for two pieces of text, two images, or two objects in general, to be similar?
 
 This is a relatively simple idea that is actually incredibly interesting and intricate once we start to dive into the details.
 
-But for now, you should know that machine learning (ML) models are key to this whole process. Similar models to those that allows clever text generation from prompts power vector searches. Instead of generating new text, here these models capture "meaning" of pieces of text or other media. We will cover this in more detail later on.
+But for now, you should know that machine learning (ML) models are the key to this whole process. Some ML models use the prompts that power vector searches to generate text responses. Instead of generating new text, the Weaviate models capture the "meaning" of text or other media. We will cover this in more detail later on.
 
 ### <i class="fa-solid fa-chalkboard"></i>&nbsp;&nbsp;Indexing (vector) data
 
-Vector searches can be very computationally intensive.
+Vector searches can be very intensive computationally.
 
-To overcome this problem, Weaviate uses a combination of indexes including an approximate nearest neighbor (ANN) index and an inverted index. They respectively allow Weaviate to perform extremely fast vector searches, as well as to filter data using Boolean criteria on data.
+To overcome this problem, Weaviate uses a combination of indexes including an approximate nearest neighbor (ANN) index and an inverted index. Together they allow Weaviate to perform extremely fast vector searches, They also let Weaviate filter data using Boolean criteria.
```

Comment:
ANN & inverted indices perform different functions: 
- ANN index -> fast vector search
- Inverted index -> filtering using Boolean criteria

So I would like to go back to the original.